template/nestJs/dockerfile

```
@@ -1,31 +1,28 @@
-FROM node:lts as BUILDER
+FROM node:alpine as BUILDER
 WORKDIR /builder/
 ADD . .
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)